terraform/aws/etc/eu-west-2_vp.tfvars

```diff
@@ -57,6 +57,20 @@ gp2gp_create_mhs_mock = true
 gp2gp_mongo_options = "replicaSet=rs0&readPreference=secondaryPreferred&retryWrites=false&tls=true"
 gp2gp_ssl_trust_store_url = "s3://nhsd-aws-truststore/rds-truststore.jks"
 
+# Settings for "lab-results" component
+lab-results_service_minimal_count = 1
+lab-results_service_desired_count = 1
+lab-results_service_maximal_count = 1
+lab-results_service_target_request_count = 1200
+lab-results_service_container_port = 8080
+lab-results_service_launch_type = "FARGATE"
+lab-results_log_level = "DEBUG"
```

Comment:
Can we set this to INFO as this will use up lots of resources, we should only be setting this to debug when required